hw/xbox/mcpx_apu.c

```diff
@@ -1620,4 +1744,34 @@ void mcpx_apu_init(PCIBus *bus, int devfn, MemoryRegion *ram)
     /* Keep pointers to system memory */
     d->ram = ram;
     d->ram_ptr = memory_region_get_ram_ptr(d->ram);
+
+    d->gp.dsp = dsp_init(d, gp_scratch_rw, gp_fifo_rw);
+    d->ep.dsp = dsp_init(d, ep_scratch_rw, ep_fifo_rw);
+    d->set_irq = false;
+    d->exiting = false;
+
+    AUD_register_card("XboxAPU", &d->qemu_audio_card);
+
+    struct audsettings as = {
+        .freq = 48000,
+        .nchannels = 1,
+        .fmt = AUDIO_FORMAT_S32,
+        .endianness = 0,
+    };
+
+    d->qemu_audio_voice_out = AUD_open_out(&d->qemu_audio_card,
+                                           d->qemu_audio_voice_out,
+            "XboxAPU.out", d, mcpx_vp_out_cb, &as);
```

Comment:
This name is not QEMU style (as far as I remember).
The QEMU names appear to be lowercase.

I chose "mcpx-apu.vp_bins" in my code for similar bins. It should also be clearly marked as debug output, as no such output exists in the hardware. APU is an audio processor, not capable of driving audio output directly (that's what the ACI does).

Reply:
`mcpx-apu.vp_bins` is better, I'll use that. Thanks!